src/clp_ffi_js/constants.hpp

```diff
@@ -19,6 +19,7 @@ enum class LogLevel : std::uint8_t {
     FATAL,
 };
```

Comment:
Can we add a `Length` as the last item and use it to replace `cValidLogLevelsEndIdx`?
```suggestion
    Length
};
```
That way we don't always have to update `cValidLogLevelsEndIdx` if we ever add more log levels.

Reply:
I was looking at https://github.com/y-scope/clp/blob/f1876cd63a56dd79a6243f6c8b6b6a7fd48348cc/components/core/src/clp/GlobalMySQLMetadataDB.cpp#L22-L23 when i suggested the enum name "length" though I don't see the convention being documented anywhere. If that's a convention, i feel we can omit the comment.

@kirkrodrigues could you confirm the convention?

Reply:
I think it's a general C/C++ convention to call it length; that said, it's probably not a bad idea to leave a comment because maybe the reader would confuse "SIZE" for a log level.

I also realized that the case of these enum values is wrong. They should be `None`, `Trace`, etc. Can we open an issue and fix in another PR?

Reply:
@coderabbit let's open an issue to correct cases of enum property names as PascalCase